cablelabs/lpwanserver
# Seperated the V1 and V2 Redis in docker to fix the incompatiblility b…

…etween device activation keys.

#### What does this PR do?
Separates the Redis url endpoints for V1 and V2 of LoRaServer in the demo/tests
#### Do you have any concerns with this PR?
No
#### How can the reviewer verify this PR?
Run ./bin/e2e-tests and (do not close docker) Open https://localhost:8081 and verify the device activiation keys exist for BobsMouseTrapV1 Device 
#### Any background context you want to provide?
Before when the device session was synched to V2 LoRa Server from V1, it changed the data fields on the session which was incompatible with V1.  This caused a server crash.  
#### Screenshots or logs (if appropriate)
#### Questions:
- Have you connected this PR to the issue it resolves?
yes #199 
- Does the documentation need an update?
no
- Does this add new dependencies?
no
- Have you added unit or functional tests for this PR?
more or less
